help/de/Help_ch_GUIMenus.xml

```diff
@@ -42,20 +42,16 @@
         <title>Das &app; Hauptfenster</title><mediaobject>
           <imageobject role="html">
             <imagedata fileref="figures/Main-window-callouts.png" format="PNG"
-                       srccredit="Christian Wehling" width="510" />
+                       srccredit="Frank H. Ellenberger" width="510" />
           </imageobject>
 
           <imageobject role="fo">
             <imagedata fileref="figures/Main-window-callouts.png" format="PNG"
-                       srccredit="Christian Wehling" />
+                       srccredit="Frank H. Ellenberger" />
           </imageobject>
 
-          <textobject>
-            <phrase>Bestandteile des &app; Hauptfensters</phrase>
-          </textobject>
-
           <caption>
-            <para>Die Elemente, in die das Hauptfenster von &app; unterteilt ist.
+            <para>Die Elemente, die zur Bedienung von &app; im Hauptfenster zur Verfügung stehen.
```

Comment:
In dem Bild geht es eigentlich erstmal um die verschiedenen **Bereiche** des Hauptfensters. Die einzelnen Elemente kommen ja später in den Tabellen.

Ich denke, das sollten wir bereits im Bildtitel deutlich machen und für den Backport markieren.
Das hat mich unterbewußt schon seit Jahren gestört, ist mir aber eben erst klar geworden